hiss/hiss/settings/base.py

```diff
@@ -81,7 +81,7 @@
 
 MEDIA_ROOT = "resumes/"
 MEDIA_URL = "/resumes/"
-
+LOGOUT_REDIRECT_URL = "/"
```

Comment:
Please use `reverse_lazy` for this. This route pattern might not always match.

```suggestion
LOGOUT_REDIRECT_URL = reverse_lazy("customauth:login")
```